salesagility/SuiteCRM
# Fixed #239 Missed icons has been introduced into all 3 themes

Load the missing icons in admin/studio module list
## Description

According to the issue, Running 7.2 that was upgraded to 7.2.1. The Modules list in Studio seem to   be missing 3 images for Index, Index Events and Locations.
The testing has been done on version 7.7  the key changes on issued screen are:
The modules  index and index events, are not exist any more in this location.
One icon from original description left  "Locations". It also introduced new module without icon: Business Hours.
For both modules has been provided icons,
the changes has been made in all available themes,
References issue  #239.


Open the admin/studio panel and see the missing icons. 



## Motivation and Context

Provide missing icons for better appearance of studio/ module list screen.

## How To Test This

Open admin/studio, check appearance of the icons for Bussiness Hours and Locations
Repeat steps in all themes
## Types of changes


### Final checklist
